src/Raven.Studio/typescript/viewmodels/database/indexes/editIndex.ts

```diff
@@ -247,6 +264,88 @@ class editIndex extends viewModelBase {
             });
     }
 
+    private fetchIndexHistory() {
+        const db = this.activeDatabase();
+        
+        return new getIndexHistoryCommand(db, this.editedIndex().name() || this.originalIndexName)
+            .execute()
+            .done((indexHistory) => {
+                const indexHistoryInfo = indexHistory.History.map(x => {
+                    return { CreatedAt: x.CreatedAt, Definition: x.Definition, Source: x.Source, CreatedAtFormatted: generalUtils.formatUtcDateAsLocal(x.CreatedAt) };
```

Comment:
any reason why we format it here - instead of having raw data and format on UI?

Reply:
+ please avoid doing things like: 
```
CreatedAt: x.CreatedAt, Definition: x.Definition, Source: x.Source
```

when we add new prop then it will break

try instead:
```
{
   ...x,
  CreatedAtFormatted: generalUtils.formatUtcDateAsLocal(x.CreatedAt)
}

Reply:
done